src/app/treasure-hunt/treasure-hunt-report/executive-summary/executive-summary.component.ts

```diff
@@ -83,22 +98,70 @@ export class ExecutiveSummaryComponent implements OnInit {
     this.dataTitles = dataTitles;
   }
 
-  getChartHeight(): number {
-    if (this.costSummaryChartContainer) {
-      return this.costSummaryChartContainer.nativeElement.clientHeight;
+  prepPieChartData() {
+    this.pieChartValues = new Array<number>();
+    this.pieChartLabels = new Array<string>();
+    for (let i = 0; i < this.teamData.length; i++) {
+      this.pieChartLabels.push(this.teamData[i].team + ' $' + (Math.round(this.teamData[i].costSavings)));
+      this.pieChartValues.push(this.teamData[i].costSavings);
+    }
+  }
+
+  prepTeamData() {
+    let teams: Array<string> = new Array<string>();
+    this.teamData = new Array<any>();
+    for (let i = 0; i < this.opportunityCardsData.length; i++) {
```

Comment:
overly complicated for loop...

![prepTeamData](https://user-images.githubusercontent.com/10789740/67117942-74a13600-f1a9-11e9-8f17-2e3244ba9294.PNG)